lxc/launch.go

```diff
@@ -4,45 +4,68 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/codegangsta/cli"
 	"github.com/lxc/lxd"
 	"github.com/lxc/lxd/shared"
-	"github.com/lxc/lxd/shared/gnuflag"
 	"github.com/lxc/lxd/shared/i18n"
 )
 
-type launchCmd struct {
-	init initCmd
+var commandLaunch = cli.Command{
+	Name:      "launch",
+	Usage:     i18n.G("Launch a container from a particular image."),
+	ArgsUsage: i18n.G("[remote:]<image> [remote:][<name>] [--ephemeral|-e] [--profile|-p <profile>...]"),
+	Description: i18n.G(`Launches a container using the specified image and name.
+
+   Not specifying -p will result in the default profile.
+   Specifying "-p" with no argument will result in no profile.`),
+
+	Flags: []cli.Flag{
```

Comment:
Here you're forgetting the global flags if I'm not mistaken. Let's make a wrapper for this :)